Adds 2.8.0 release notes (#145)
ondat/documentation: docs/release-notes.md

To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.

## 2.8.0 - Release 2022-06-29

2.8.0 Release Notes

### New

k8s

* Etcd in Production: We have added support for putting ETCD in your cluster in a production environment
* Modified CSI provisioner to work with Snapshots

> * Ondat volumes metrics exporter: we have added support for Prometheus Endpoint to allow users to view metrics on Ondat Volumes

Reviewer: Small nit:

```suggestion
* Ondat volumes metrics exporter: we have added a Prometheus endpoint to allow users to view metrics for Ondat Volumes
```
